src/main/java/io/dev/jobprep/exception/CustomAccessDeniedHandler.java

```diff
@@ -0,0 +1,40 @@
+package io.dev.jobprep.exception;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dev.jobprep.exception.dto.ErrorResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.web.access.AccessDeniedHandler;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+import static io.dev.jobprep.exception.code.ErrorCode403.AUTH_ACCESS_DENIED;
+
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class CustomAccessDeniedHandler implements AccessDeniedHandler {
+    private final ObjectMapper objectMapper;  // 추가 필요
+    @Override
+    public void handle(HttpServletRequest request,
+                       HttpServletResponse response,
+                       AccessDeniedException accessDeniedException) throws IOException {
+
+        log.info("Access Denied Exception: {}, Path: {}",
+                accessDeniedException.getMessage(),
+                request.getRequestURI());
+
+        response.setContentType("application/json");
```

Comment:
@GlobalExceptionHandler에 보면 `AccessDeniedException` 을 처리하는 핸들러를 구현해두었습니다. 굳이 예외 처리 로직을 직접 작성할 필요 없이 Spring에서 제공하는 resolver를 사용해서 예외를 처리하면 될 것 같습니다~!

```suggestion
        resolver.resolveException(request, response, null, accessDeniedException);
```